src/pybind/mgr/dashboard_v2/tests/test_nodb.py

```diff
@@ -182,7 +182,7 @@ def get_all_objects(api_controller, query):
             my_dict = JsonField(base_type=list, primary_key=True)
 
         class DictFieldModelSerializer(NodbSerializer):  # pylint: disable=E0602
-            class Meta:
+            class Meta(object):
```

Comment:
If you need to fix code in a skipped test, can you instead please remove this test altogether?